### DISCUSSION TITLE
Possible to create a player class from the pick class from draft picks?

### DISCUSSION THREAD

**Discussion by ntellakula** [cwendt94/espn-api#506] [ESPN API] [answered]
`league.draft` allows me to grab the League's draft, but each pick and player only returns player IDs and names. is there a way to acquire other information of the player, such as team and position, through the angle of the draft and picks?

the goal is to count WRs, RBs, etc. drafted by each player

**Comment by cwendt94** [comment, marked as answer]
So the draft API response only gives that information back. There are two ways you then could map that player to a player class. One the draft does have the Team class associated with it so you could filter through the team roster and find the player matching the playerId. However if that player is no longer on the Team it could be an issue, so you could use the `league.player_info(playerId='123')` and pass the playerId from the draft. That will return the player class.

**Reply by ntellakula** [reply]
figured as much - thanks, will move forward with this
